README.md

```diff
@@ -45,6 +45,16 @@ python -m get_cover_art [--path=<path_to_audio_file_or_folder>] [--test] [--othe
 
   --path PATH           audio file, or folder of audio files (recursive)
   --dest DEST           destination of artwork
+  --use-folder-art {before,after,none}
+                        Use image from local folder; "before" prevents
+                        downloads, "after" uses as a fallback.
+  --folder-art-name FOLDER_ART_NAME [FOLDER_ART_NAME ...]
+                        Filename(s) of folder art to use. Accepts {artist},
+                        {album}, and {title} for replacement: e.g. cover.jpg
+                        or {album}-{artist}.jpg
+  --output-filename OUTPUT_FILENAME
```

Comment:
I would suggest renaming this to art-filename / ART_FILENAME, because "output" could be misconstrued to mean the audio files.